docker-compose-gpu.yml

```diff
@@ -46,4 +50,5 @@ services:
     environment:
       - API_ENDPOINT=http://haystack-api:8000
       - EVAL_FILE=eval_labels_example.csv
+      - HAYSTACK_UI_DISABLE_FILE_UPLOAD=1
```

Comment:
For now ok, but if we see any further divergence between public demo and the "local quick start", we should think about splitting them somehow. Users might want to see the file upload locally and are not aware of changing this setting here